10/umbraco-deploy/release-notes.md

```diff
@@ -32,33 +32,23 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
 *   Fixed display child nodes indicator for tree picker used for selecting items in the remote environment [#146](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/146)
 *   Added additional logging to indicate which item and pass causes a processing failure if and when one occurs [#144](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/144)
 *   Tidied up initialization markers [#102](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/102)
+  
+</details>
 
-### Deploy Contrib 9.1.1, 10.1.1 and 11.0.1 (February 14th 2023)
+<details>
 
-*   Block grid editor support and fix for null reference exception (10+)
-*   Further caching for deploy operations.
+<summary>Umbraco Deploy Version 10</summary>
```

Comment:
It was actually these headings I was referring to 😅 

```suggestion
<summary>Version 10</summary>
```

Reply:
Ahhh okay, I've removed it.